src/components/operations/operation-details/ko/runtime/graphql-console.ts

```diff
@@ -681,6 +743,12 @@ export class GraphqlConsole {
 
     private editorValidations(): void {
         this.editorErrors([]);
+
+        const hasWsProtocol = !!this.api().protocols.find(p => p == "ws" || p == "wss");
+        if (this.isSubscriptionOperation() && !hasWsProtocol) {
+            this.editorErrors.push(`Websocket protocol not enabled, you can enable it in Azure > APIM > APIs > ${this.api().name} > Settings tab, and reload Developer Portal.`);
```

Comment:
@javierbo1989, the dev portal is for C2 customers, they won't be able to go to Azure portal. How about something like "Live subscriptions are not supported for this API". @mikebudzynski can you please help with messaging?